### PR TITLE
[examples] fix: ant-design unstyled hydration issue

### LINKED ISSUE #48483: Flash of Unstyled content using Ant Design with Nextjs
### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

```bash
Operating System:
      Platform: darwin
      Arch: x64
      Version: Darwin Kernel Version 22.2.0: Fri Nov 11 02:08:47 PST 2022; root:xnu-8792.61.2~4/RELEASE_X86_64
    Binaries:
      Node: 16.15.0
      npm: 8.5.5
      Yarn: 1.22.18
      pnpm: N/A
    Relevant packages:
      next: 13.3.1-canary.8
      eslint-config-next: 13.3.0
      react: 18.2.0
      react-dom: 18.2.0
```

### Which example does this report relate to?

with-ant-design

### What browser are you using? (if relevant)

Chrome 112.0.5615.49	

### How are you deploying your application? (if relevant)

Vercel

### Flash of Unstyled content using Ant Design with Nextjs :
When using Ant Design with Next JS 
the HTML gets rendered unstyled ( barebone HTML elements) for a fraction of a second , and then the styles get loaded and the components are rendered correctly

### Expected Behavior

the first contentful-paint should be properly styled

### To Reproduce

clone the with-ant-design example and host it

### PR DESCRIPTION
**Pull request by NavidNourani**
### What?
Fix flash of Unstyled content using Ant Design with Nextjs.

### Why?
There was an open issue on the GitHub issues list.

### How?
Just upgrading an And-Design package.

Fixes #48483

### CODE REVIEW

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: 100fbbf1a5388e4afa53bbaf6a92e9b615e62f22

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

**Comment by NavidNourani** [comment]
Some jobs failed and I cannot understand where the problem is.

File: examples/with-ant-design/package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "start": "next start"
   },
   "dependencies": {
-    "@ant-design/cssinjs": "^1.16.1",
+    "@ant-design/cssinjs":  ">=1.21.0",
```
Comment (Juneezee) [inline examples/with-ant-design/package.json:9]: There is an extra space

```suggestion
    "@ant-design/cssinjs": ">=1.21.0",
```

Comment (NavidNourani) [reply examples/with-ant-design/package.json]: Thank you so much!
Fixed!

**Review by leerob — APPROVED** [review]

**Pull request merged**